studyzen/FlashCard/FlashCardController.cs

```diff
@@ -0,0 +1,110 @@
+using Microsoft.AspNetCore.Mvc;
+using StudyZen.Common;
+using StudyZen.FlashCards.Requests;
+using Microsoft.AspNetCore.Mvc.ApiExplorer;
+
+
+namespace StudyZen.FlashCards;
+
+[ApiController]
+[Route("[controller]")]
+public sealed class FlashCardsController : ControllerBase
+{
+    private readonly IFlashCardService _flashCardService;
+
+    public FlashCardsController(IFlashCardService flashCardService)
+    {
+        _flashCardService = flashCardService;
+    }
+
+    [HttpPost]
+    public IActionResult CreateFlashcard([FromBody] CreateFlashCardRequest? request)
+    {
+        request = request.ThrowIfRequestArgumentNull(nameof(request));
+
+        var flashCardId = _flashCardService.AddFlashCard(request);
+
+        var response = new
+        {
+            FlashCardSetId = request.FlashCardSetId,
+            FlashCardId = flashCardId,
+            Question = request.Question,
+            Answer = request.Answer
+
+        };
+
+        return CreatedAtAction(nameof(GetFlashcard), new { flashCardId = flashCardId }, response);
+    }
+
+    [HttpGet]
+    [Route("{flashCardId}")]
+    public IActionResult GetFlashcard(int flashCardId)
+    {
+        var flashCard = _flashCardService.GetFlashCard(flashCardId);
+        if (flashCard == null)
```

Comment:
change the if/else statements to the ternary operator to save some code lines return flashCard == null ? NotFound() : Ok(flashCard);

Reply:
Fixed